examples/finegrained_plot.py

```diff
@@ -0,0 +1,76 @@
+"""
+===============================
+Fine grained Plotting Features
+===============================
+
+An example to show control over various plotting features.
+"""
+###############################################################################
+# Import the necessary modules for plotting.
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+
+###############################################################################
+# SkyCoord module provides flexible celestial coordinate representation and a
+# draw_limb method overplots the boundary of the spacial objects. Date of the
```

Comment:
Maybe its due to a lack of my second coffee, but I am confused by the meaning of, `boundary of the spacial objects`